### DISCUSSION TITLE
mark as read is not working

### DISCUSSION THREAD

**Discussion by King-GO** [community/community#54210] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

![mark as read](https://user-images.githubusercontent.com/54804407/235562225-5874471e-f806-4b72-9a74-a7d467c9c833.jpg)

tested on Chrome and Edge

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @King-GO ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by King-GO** [comment]
It's not easy to find the right place to report the issue, could you give me the link?

**Reply by radar-blip** [reply]
You can report it here: [Bug-Report](https://support.github.com/contact/bug-report)
